### PR TITLE
CIV-10916 CYA: How do you wish to proceed

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-10916

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/services/features/claimantResponse/claimantResponseTasklistService/claimantResponseTasks/whatToDoNextSectionTasks.ts [hunk 1]
```diff
@@ -134,8 +134,11 @@ export function getProposeAlternativeRepaymentTask(claim: Claim, claimId: string
     url: constructResponseUrlWithIdParams(claimId, CLAIMANT_RESPONSE_PAYMENT_OPTION_URL),
     status: TaskStatus.INCOMPLETE,
   };
-  if ((claim.respondent1.type === 'ORGANISATION' || claim.respondent1.type === 'COMPANY')
-     || claim.claimantResponse?.suggestedPaymentIntention?.paymentOption !== undefined) {
+
+  if ((claim.isPAPaymentOptionPayImmediately() && claim.claimantResponse?.courtProposedDate?.decision)
```
Comment (jarekPierchala) [inline src/main/services/features/claimantResponse/claimantResponseTasklistService/claimantResponseTasks/whatToDoNextSectionTasks.ts:138]: codition to separate method please

**Review by jarekPierchala — CHANGES_REQUESTED** [review]

File: src/main/services/features/claimantResponse/checkAnswers/checkAnswersService.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ const logger = Logger.getLogger('claimantResponseCheckAnswersService');
 
 const buildSummarySections = (claimId: string, claim: Claim, lang: string, claimFee?: number): SummarySections => {
   const getYourResponseSection = () => {
-    return claim.isFullDefence() || claim.isPartialAdmission()
+    return claim.isFullDefence() || claim.isPartialAdmission() || claim.isFullAdmission()
```
Comment (GorilaNaranja) [inline src/main/services/features/claimantResponse/checkAnswers/checkAnswersService.ts:21]: I'm not sure but here we are going to call always buildYourResponseSection right?
If is full defence, is partial admission or is full admission cover all the possibilities if I'm not wrong. So maybe we don't need this conditions and we can return directly buildYourResponseSection()

Comment (ElifDenizEsea) [reply src/main/services/features/claimantResponse/checkAnswers/checkAnswersService.ts:21]: I think they're being added one by one

**Review by jarekPierchala — APPROVED** [review]

**Comment by jarekPierchala** [comment]
Will be part of post-case-progression-tasks branch

**Pull request closed without merging**